.github/workflows/release.yml

```
@@ -22,9 +22,33 @@ jobs:
       - name: Run tests
         run: cargo test --verbose
 
-  docker:
+  compile:
     needs: [test]
-    if: github.repository == 'tensorchord/Horust'
+    # if: github.repository == 'tensorchord/Horust'
```

To avoid accident release by forked repo.
```suggestion
    if: github.repository == 'tensorchord/Horust'
```

---

> set 755 permission(test)

Just for a test, I'm releasing it to my dockerhub to check if it works.

---

I will reset the git history to overwrite this test commit.

---

Maybe it's better to test on another branch checkouted from this one.